components/script/dom/webglrenderingcontext.rs

```diff
@@ -3805,48 +3805,50 @@ impl UniformSetterType {
     }
 }
 
-#[derive(Default, JSTraceable, MallocSizeOf)]
+#[derive(JSTraceable, MallocSizeOf)]
 #[must_root]
-pub struct BoundAttribBuffers {
-    elements: DomRefCell<FnvHashMap<u32, (bool, Option<Dom<WebGLBuffer>>)>>,
+pub struct VertexAttribs {
+    attribs: DomRefCell<Box<[(bool, Option<Dom<WebGLBuffer>>)]>>,
```

Comment:
It may be worth a comment here mentioning that the slice contains as many attribute buffers as possible, that the `bool` represents whether it's enabled as an array (should it be converted to a binary `enum`?), and that the buffer can be `None` when removed.